pkg/controllers/uiplugin/troubleshooting_panel.go

```diff
@@ -29,33 +31,23 @@ func createTroubleshootingPanelPluginInfo(plugin *uiv1alpha1.UIPlugin, namespace
 		extraArgs = append(extraArgs, fmt.Sprintf("-features=%s", strings.Join(features, ",")))
 	}
 
-	proxyName, proxyNamespace := "korrel8r", "korrel8r"
-
-	if plugin.Spec.TroubleshootingPanel != nil {
-		if plugin.Spec.TroubleshootingPanel.Korrel8r.Name != "" {
-			proxyName = plugin.Spec.TroubleshootingPanel.Korrel8r.Name
-		}
-		if plugin.Spec.TroubleshootingPanel.Korrel8r.Namespace != "" {
-			proxyNamespace = plugin.Spec.TroubleshootingPanel.Korrel8r.Namespace
-		}
-	}
-
 	pluginInfo := &UIPluginInfo{
 		Image:             image,
 		Name:              plugin.Name,
 		ConsoleName:       "troubleshooting-panel-console-plugin",
 		DisplayName:       "Troubleshooting Panel Console Plugin",
 		ResourceNamespace: namespace,
+		LokiServiceNames:  make(map[string]string),
 		ExtraArgs:         extraArgs,
 		Proxies: []osv1alpha1.ConsolePluginProxy{
 			{
 				Type:      osv1alpha1.ProxyTypeService,
 				Alias:     "korrel8r",
 				Authorize: false,
 				Service: osv1alpha1.ConsolePluginProxyServiceConfig{
-					Name:      proxyName,
-					Namespace: proxyNamespace,
-					Port:      8443,
+					Name:      "korrel8r",
```

Comment:
Just in case the name might change at some point, I'd put this back as a var.

Reply:
Done